eos/db/saveddata/fit.py

```diff
@@ -106,7 +106,7 @@ def __repr__(self):
                collection_class=HandledProjectedModList,
                cascade='all, delete, delete-orphan',
                single_parent=True,
-               primaryjoin=and_(modules_table.c.fitID == fits_table.c.ID, modules_table.c.projected == True)),
+               primaryjoin=and_(modules_table.c.fitID == fits_table.c.ID, modules_table.c.projected is True)),
```

Comment:
@Ebag333 I must have gotten the order of the commits confused then. I thought you had fixed these, and then did this commit which reverted the fix haha

No worries with this then :)